### PR TITLE
IPRO_18:Home page added

### PR DESCRIPTION
**Pull request by MbiydzenyuyStephen**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 function App() {
   return (
     <Router>
         <Switch>
             <Route exact path="/"
-            component={Signin} />
-            <Route  path="/signup"
-            component={Signup} />
+            component={SignIn} />
+            <Route  path="/SignUp"
```
Comment (Lambertyubin) [inline src/App.js:14]: paths should be in lowercase. eg `/signup`

File: src/App.js [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+import SignUp from './components/signup/SignUp';
+import SignIn from './components/signin/SignIn';
+import Home from './components/home/Home';
 
 
 function App() {
   return (
     <Router>
         <Switch>
             <Route exact path="/"
```
Comment (Lambertyubin) [inline src/App.js:12]: This `/` route should lead to the home page

File: src/components/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import NavBar from "../navbar/NavBar";
+import { FaSistrix } from "react-icons/fa";
+import "./home.css";
+
+const Home = () => {
+  return (
+    <div className="parent-div">
+      <NavBar />
+      
+      <div className="search-section">
```
Comment (Lambertyubin) [inline src/components/home/Home.jsx:11]: Extract this into a new component that could call `SearchSection`

**Review by lambertydev — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (MbiydzenyuyStephen) [reply src/App.js:14]: Done

**Review by lambertydev — CHANGES_REQUESTED** [review]

File: src/components/home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+              <SecondCard />
+            </div>
+            <div className="col-12 col-lg-4">
+              <SecondCard />
+            </div>
+            <div className="col-12 col-lg-4">
+              <SecondCard />
+            </div>
+          </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (MbiydzenyuyStephen) [reply src/components/home/Home.jsx:41]: Ok Sir only from line 14 to line 28 is related to search

File: src/components/home/Home.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import NavBar from "../navbar/NavBar";
+import "./home.css";
+import SearchSection from "../searchsection/SearchSection";
+import Card from "../card/card";
+import SecondCard from "../secondcard/SecondCard";
```
[the remark this answers is not in the document]
Comment (MbiydzenyuyStephen) [reply src/components/home/Home.jsx:6]: ok Sir
I will rename it as static card

File: src/components/staticcard/StaticCard.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import StaticCard1 from "../../assets/static_card_image1.png";
+import StaticCard2 from "../../assets/static_card_image2.png";
+import StaticCard3 from "../../assets/static_card_image3.png";
+import "./staticcard.css";
+
+const StaticCard = () => {
```
Comment (Lambertyubin) [inline src/components/staticcard/StaticCard.jsx:8]: StaticCard isn't a good name. You could just name it `PropertyCard`.

File: src/components/staticcard/StaticCard.jsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import StaticCard1 from "../../assets/static_card_image1.png";
+import StaticCard2 from "../../assets/static_card_image2.png";
+import StaticCard3 from "../../assets/static_card_image3.png";
```
Comment (Lambertyubin) [inline src/components/staticcard/StaticCard.jsx:5]: StaticCard1, StaticCard2, etc doesn't make sense. They bring confusion between your `StaticCard` component and images.

File: src/components/staticcard/StaticCard.jsx [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+                  <p className="card-text">
+                    We’re creating a seamless online experience – from shopping
+                    on the largest rental network, to applying, to paying rent.
+                  </p>
+                  <div className="static-card-button">
+                    <Button className="btn btn-link">Find rentals</Button>
+                  </div>
+                </div>
+              </div>
+            </div>
```
Comment (Lambertyubin) [inline src/components/staticcard/StaticCard.jsx:32]: This piece of code seems to be repeated in this component. Can you extract it into a separate component and pass the values that it needs through props?
For example, the first one could have a `title` prop set to "Buy a house",  the second one's title prop will have value "Sell a house", etc. Other props could include: "description", "callToActionButtonText", and "image"

**Review by lambertydev — CHANGES_REQUESTED** [review]
Where ever you see that code is duplicated, you should extract it into a new component that you can reuse in different places in your code to avoid duplication.

File: src/components/staticcard/StaticCard.jsx (hunk 7 above)
Comment (MbiydzenyuyStephen) [reply src/components/staticcard/StaticCard.jsx:5]: ok Sir

File: src/components/propertycard/PropertyCardParent.jsx [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+      <PropertyCardChild object={buypropertycard} />
+
+      //Sir lines 24 and 25 are duplicating in the browser 
+    </div>
+  );
+};
+
+export default PropertyCardParent;
+
+// <div className="container">
```
Comment (Lambertyubin) [inline src/components/propertycard/PropertyCardParent.jsx]: you should remove the comment before merging

**Review by lambertydev — APPROVED** [review]
I'm approving any PR that still has to do with layout. I'm looking forward to review PRs where you implement functionality.

**Pull request merged**